frontend/common.ts

```diff
@@ -63,8 +76,8 @@ export interface MapshotJSON {
     // Maximal available zoom level index (most detailed)
     zoom_max: number,
 
-    // Current position of the player.
```

Comment:
The frontend code can be used on snapshots made before this change, so I would prefer to keep some backward compatibility. In practice:
 - In the .lua code, only generate the new field (`players`).
 - Here, parse both the `player` and `players` fields.
 - In viewer.ts, keep both code paths, for `player` and `players` (only one should be generated, so conflicts are unlikely and minor).

Reply:
Done and done. Out of curiosity, at what point would you be comfortable with breaking backwards compatibility?

Reply:
So far my approach has been "keep backward compatibility as long as it is easy to do; otherwise try to break in a not-so-terrible way".